frontend/src/pages/forms/Form.jsx

```diff
@@ -1,20 +1,19 @@
-import React, { useState } from 'react';
-import TokenSelector from 'components/TokenSelector';
+import { ReactComponent as AlertHexagon } from 'assets/icons/alert_hexagon.svg';
 import BalanceCards from 'components/BalanceCards';
+import CardGradients from 'components/CardGradients';
+import CongratulationsModal from 'components/congratulationsModal/CongratulationsModal';
+import PositionModal from 'components/modals/Modal';
```

Comment:
you are importing components/modals/Modal, but your file is named `modal` so it should be components/modals/modal 

Currently it will fail to build

Reply:
And in general, you should give more meaningful name to a file, ClosePositionModal (e.g.)